system/modules/core/controllers/BackendMain.php

```diff
@@ -176,95 +260,4 @@ protected function welcomeScreen()
 
 		return $objTemplate->parse();
 	}
-
-
-	/**
-	 * Output the template file
-	 */
-	protected function output()
```

Comment:
This is not a good idea, I'm pretty sure there are people using `BackendMain` as there parent class to override stuff like that.